src/vs/workbench/contrib/comments/browser/commentNode.ts

```diff
@@ -125,6 +125,7 @@ export class CommentNode<T extends IRange | ICellRange> extends Disposable {
 		this.createHeader(this._commentDetailsContainer);
 		this._body = document.createElement(`div`);
 		this._body.classList.add('comment-body', MOUSE_CURSOR_TEXT_CSS_CLASS_NAME);
+		this._body.setAttribute('style', 'max-height: 20em;');
```

Comment:
The maximum height is currently hardwired, but I wondered about adding it as a property of `CommentOptions` so a comment controller could specify it.

Reply:
Can this be added in https://github.com/gjsjohnmurray/vscode/blob/pr/gjsjohnmurray/180044/src/vs/workbench/contrib/comments/browser/media/review.css instead of inline?

Reply:
Done. I had inlined it in case we wanted to make it settable.